src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleDeclarations = sourceString.split(';');
```

Comment:
When using the 'split' method, it's a good practice to trim each declaration to ensure there's no leading or trailing whitespace. This can be done by chaining the 'map' method after 'split'.

Reply:
You can also use trim here